sources/osg/RenderStage.js

```diff
@@ -71,6 +71,17 @@ define( [
         getCamera: function () {
             return this.camera;
         },
+        getPositionnedAttribute: function ( node ) {
```

Comment:
Seems better, just add the accessors to return the positioned attribute. We will put the find inside shadowMap for now.